airgun/views/discoveredhosts.py

```diff
@@ -232,8 +230,7 @@ class DiscoveredHostProvisionDialog(BaseLoggedInView):
 
     @property
     def is_displayed(self):
-        return self.browser.wait_for_element(
-            self.title, exception=False) is not None
+        return self.title.is_displayed
```

Comment:
What's the reason behind this change? It will probably lead to exceptions in logs when element is not found

Reply:
It will not leed to exception and working as expected, the older implementation was not working always failing when calling is_displayed

Reply:
now the tests are very robust in this area, and never fail any more. locally or on Saucelabs.

Reply:
Sorry I confused with an other case, the problem here was that it navigated to all discovered hosts and the navigation stopped, as the older implementation was returning always True.

Reply:
We use `is_diplayed` just like that directly for `welcome_message` in searching, and i see pretty often messages like that in logs:
```
2018-09-27 17:52:20 - widgetastic_null - ERROR - [ContentViewVersionDetailsView/errata/welcome_message]: An exception happened during read call (elapsed 33 ms)
2018-09-27 17:52:20 - widgetastic_null - ERROR - [ContentViewVersionDetailsView/errata/welcome_message]: Message: Could not find an element Locator(by='xpath', locator="//div[@class='blank-slate-pf' or @id='welcome']")
Traceback (most recent call last):
  File "/Users/andrii/workspace/widgetastic.core/src/widgetastic/browser.py", line 335, in element
    return elements[0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/andrii/workspace/widgetastic.core/src/widgetastic/log.py", line 117, in wrapped
    result = f(self, *args, **kwargs)
  File "/Users/andrii/workspace/widgetastic.core/src/widgetastic/widget.py", line 1064, in read
    return self.text
  File "/Users/andrii/workspace/widgetastic.core/src/widgetastic/widget.py", line 1061, in text
    return self.browser.text(self, parent=self.parent)
  File "/Users/andrii/workspace/widgetastic.core/src/widgetastic/browser.py", line 595, in text
    text = self.element(locator, *args, **kwargs).text
  File "/Users/andrii/workspace/widgetastic.core/src/widgetastic/browser.py", line 327, in element
    elements = self.elements(locator, *args, **kwargs)
  File "/Users/andrii/workspace/widgetastic.core/src/widgetastic/browser.py", line 818, in elements
    force_check_safe=force_check_safe)
  File "/Users/andrii/workspace/widgetastic.core/src/widgetastic/browser.py", line 250, in elements
    locator = self._process_locator(locator)
  File "/Users/andrii/workspace/widgetastic.core/src/widgetastic/browser.py", line 203, in _process_locator
    return locator.__element__()
  File "/Users/andrii/workspace/widgetastic.core/src/widgetastic/widget.py", line 78, in wrapped
    return method(self, *new_args, **new_kwargs)
  File "/Users/andrii/workspace/widgetastic.core/src/widgetastic/widget.py", line 353, in __element__
    return self.parent_browser.element(locator)
  File "/Users/andrii/workspace/widgetastic.core/src/widgetastic/browser.py", line 337, in element
    raise NoSuchElementException('Could not find an element {}'.format(repr(locator)))
selenium.common.exceptions.NoSuchElementException: Message: Could not find an element Locator(by='xpath', locator="//div[@class='blank-slate-pf' or @id='welcome']")
```

Reply:
@abalakh it's just logged or the test fail at that line